src/saddle_point.jl

```diff
@@ -351,6 +391,11 @@ mutable struct RestartParameters
   """
   primal_weight_update_smoothing::Float64
 
+  """
+  Use an approximate localized duality gap for restart computations. The
```

Comment:
This seems to be part of a different PR.

Reply:
Moved to another PR